pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -160,9 +160,14 @@ func (c *Cluster) Start() error {
 				return errors.Wrapf(err, "failed to create mgr deployment %s", resourceName)
 			}
 			logger.Infof("deployment for mgr %s already exists. updating if needed", resourceName)
-
-			if err := updateDeploymentAndWait(c.context, c.clusterInfo, d, config.MgrType, mgrConfig.DaemonID, c.spec.SkipUpgradeChecks, false); err != nil {
+			if err := updateDeploymentAndWait(c.context, c.clusterInfo, d, config.MgrType, mgrConfig.DaemonID, c.spec.SkipUpgradeChecks, c.spec.ContinueUpgradeAfterChecksEvenIfNotHealthy); err != nil {
 				logger.Errorf("failed to update mgr deployment %q. %v", resourceName, err)
+				if c.spec.ContinueUpgradeAfterChecksEvenIfNotHealthy {
```

Comment:
With future PRs, we should add similar flow to other places too https://github.com/rook/rook/blob/ee14ecb5a79098cb98d5bd3db75b57bc169d5e74/pkg/operator/ceph/object/rgw.go#L185

Reply:
right. That's a good idea fore future PRs! ... 😄